src/appSlice.ts

```diff
@@ -0,0 +1,73 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
```

Comment:
This file is equivalent to everything inside of
- `actions`
- `reducers`
- `constants` 

and therefore removes a lot of boilerplate code